webapp/sources/pom.xml

```diff
@@ -476,15 +476,15 @@ limitations under the License.
       We declare cats in "test" here, because it is not directly needed
       in any project before rudder.
     -->
-    <specs2-version>4.20.3</specs2-version>
+    <specs2-version>4.20.5</specs2-version>
     <junit-version>4.13.2</junit-version>
     <cats-version>2.10.0</cats-version>
     <doobie-version>1.0.0-RC5</doobie-version>
-    <fs2-version>3.3.0</fs2-version>
+    <fs2-version>3.10.2</fs2-version>
     <shapeless-version>2.3.10</shapeless-version>
-    <cats-effect-version>3.5.2</cats-effect-version>
-    <dev-zio-version>2.0.19</dev-zio-version>
-    <zio-cats-version>23.1.0.0</zio-cats-version> <!-- gives fs2 3.1.6, but doobie 1.0.0-RC1 is in 3.0.3 -->
+    <cats-effect-version>3.5.4</cats-effect-version>
+    <dev-zio-version>2.0.22</dev-zio-version>
+    <zio-cats-version>23.1.0.2</zio-cats-version> <!-- gives fs2 3.10.2, but doobie 1.0.0-RC5 is in 3.9.3 -->
```

Comment:
fs2 : 3.3.0 -> 3.10.2 the update of the library does seem to have been forgotten, although cats-effect [specifies](https://github.com/typelevel/cats-effect/releases/tag/v3.5.4) it needs fs2  3.7.0+ since CE 3.5.0. As far as we know this has been working fine since we updated to 3.5.0 though...